python/ray/llm/_internal/serve/config_generator/utils/text_completion.py

```diff
@@ -0,0 +1,113 @@
+import os
+from functools import cache
+from typing import Any, Dict, Optional, Union
+
+import yaml
+
+from ray.llm._internal.serve.config_generator.utils.constants import (
+    DEFAULT_DEPLOYMENT_CONFIGS_FILE,
+    TEMPLATE_DIR,
+)
+from ray.llm._internal.serve.config_generator.utils.gpu import (
+    GPUType,
+    DEFAULT_MODEL_ID_TO_GPU,
+)
+from ray.llm._internal.serve.config_generator.utils.models import (
+    TextCompletionModelConfig,
+)
+from ray.llm._internal.serve.configs.server_models import LLMConfig
+
+
+def get_model_default_config(model_id: str) -> Dict[str, Any]:
+    return {
+        "model_loading_config": {
+            "model_id": model_id,
+        },
+    }
+
+
+@cache
+def get_default_deployment_configs(model_id: str, gpu_type: GPUType) -> LLMConfig:
+    file_path = os.path.join(TEMPLATE_DIR, DEFAULT_DEPLOYMENT_CONFIGS_FILE)
+    with open(file_path, "r") as stream:
+        configs = yaml.safe_load(stream)
+    return LLMConfig.model_validate(
+        configs["model_id_to_gpu_deployment_configs"][model_id][gpu_type.value]
+    )
```

Comment:
```suggestion
    ).deployment_config
```

Also, update the return type hint to `Dict[str, Any]`